aospy/test/aospy_test_objs/aospy_user/variables/__init__.py

```diff
@@ -0,0 +1,967 @@
+"""Collection of aospy.Var objects for use in my research."""
```

Comment:
There should be only a few variables.  One 2D, one 3D, one at phalf, one at pfull, one that's derived from one or more others, etc.